modules/fluid_structure_interaction/include/kernels/AcousticInertia.h

```diff
@@ -0,0 +1,40 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "TimeKernel.h"
+#include "Material.h"
+
+// Forward Declarations
+class TimeIntegrator;
+
+class AcousticInertia : public TimeKernel
+{
+public:
+  static InputParameters validParams();
+
+  AcousticInertia(const InputParameters & parameters);
+
+protected:
+  virtual Real computeQpResidual() override;
+
+  virtual Real computeQpJacobian() override;
+
+private:
+  const MaterialProperty<Real> & _inv_co_sq;
+  const VariableValue * _u_dot_factor;
+  const VariableValue * _u_dotdot_factor;
+  const VariableValue * _u_dot_old;
+  const VariableValue * _du_dot_du;
+  const VariableValue * _du_dotdot_du;
+
+  /// The TimeIntegrator
+  TimeIntegrator & _time_integrator;
```

Comment:
make `const` if possible

Reply:
Seems to be possible, as both `uDotFactorTag ` and `uDotDotFactorTag ` are marked `const`. Som, please make sure you address all comments left in a PR.